### PR TITLE
add new jokers and new card back; add license info

### PR DESCRIPTION
**Pull request by tsankuanglee**
## Joker cards

Adding two new joker images as alternatives to the existing two. The main motivation is to provide better visibility when the cards are stacked with a small offset.

New:
image
Original:
image

These new cards also try to provide a closer art style to the rest of the cards (while keeping the artistic convention that joker cards are supposed to be, well, jokes).

## New card back

The new card back provides a more obvious "card look".

### CODE REVIEW

**Comment by tsankuanglee** [comment]
Reviewers, please help double check the license validity. Thanks!

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1560/pr-test (or any other room on that server)

After merging, a backup will be available at `https://beta.virtualtabletop.io/editor/PR1560-pr-test`.

**Review by 96LawDawg — APPROVED** [review]
Seems like a good idea to me.  The license looks good.  It is slightly different in execution.  Other licenses are in room.html (for our default stuff) or in the JSON attribution property.  But this works as far as I'm concerned.

**Review by RaphaelAlvez — CHANGES_REQUESTED** [review]
This should be added to the decks in add widget overlay otherwise, people will not use them. They are indeed an improvement. 

What you need to change is around line 475 of editmode.js

**Comment by tsankuanglee** [comment]
> This should be added to the decks in add widget overlay otherwise, people will not use them. They are indeed an improvement.
> 
> What you need to change is around line 475 of editmode.js

Good point. Thanks for the pointer. Done.

**Comment by tsankuanglee** [comment]
working on TestCafe

File: assets/cards-default/3B.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 88.044h2.1496l-4.0211 7.9864h-2.2027l4.0739-7.9864z" fill="none" stroke="#17067a" stroke-miterlimit="10" stroke-width=".46605"/><path d="m28.982 87.951h2.1503l-4.0214 7.9871h-2.2025l4.0732-7.9871z" fill="#fff" fill-rule="evenodd" stroke-width=".46608"/><path d="m28.982 87.951h2.1503l-4.0214 7.9871h-2.2025l4.0732-7.9871z" fill="none" stroke="#17067a" stroke-miterlimit="10" stroke-width=".46605"/><path d="m22.974 88.044h-2.1503l4.0214 7.9861h2.2029l-4.0735-7.9861z" fill="#313390" fill-rule="evenodd" stroke-width=".46608"/><path d="m22.974 88.044h-2.1503l4.0214 7.9861h2.2029l-4.0735-7.9861z" fill="none" stroke="#17067a" stroke-miterlimit="10" stroke-width=".46605"/><path d="m24.879 87.965h2.1503l-4.0211 7.9867h-2.2034l4.0739-7.9867z" fill="#fff" fill-rule="evenodd" stroke-width=".46608"/><path d="m24.879 87.965h2.1503l-4.0211 7.9867h-2.2034l4.0739-7.9867z" fill="none" stroke="#17067a" stroke-miterlimit="10" stroke-width=".46605"/></g><path d="m72.822 97.338c0 1.3817-1.0548 2.515-2.3486 2.515h-52.244c-1.2903 0-2.3451-1.1333-2.3451-2.515v-83.623c0-1.3835 1.0548-2.5148 2.3451-2.5148h52.244c1.2938 0 2.3486 1.1313 2.3486 2.5148z" fill="none" stroke="#000" stroke-width=".17397"/></g></svg>
```
[the remark this answers is not in the document]
Comment (tsankuanglee) [reply assets/cards-default/3B.svg:1]: Per our conversation on Discord, we'll use this one for now and replace it when we have a better version in the future.

**Comment by tsankuanglee** [comment]
This PR is ready to go @RaphaelAlvez .

Per our conversation in Discord, we'll leave the card back image as is. Not sure how to resolve requested change here -- the latest commit includes the Deck Editor changes already.

**Comment by ArnoldSmith86** [comment]
Was there a discussion on Discord about _replacing_ 1J and 2J? I would have thought that we would add the new ones as 3J and 4J.

**Comment by 96LawDawg** [comment]
> Was there a discussion on Discord about _replacing_ 1J and 2J? I would have thought that we would add the new ones as 3J and 4J.

Yes.  3-4 people said replace.  Here:  https://discord.com/channels/770758631146782780/770758631146782783/1069402520679632986.

**Pull request merged**